samples/word/01-basics/basic-api-call.yaml

```diff
@@ -1,7 +1,7 @@
 order: 1
 id: word-basics-basic-api-call
 name: Basic API call
```

Comment:
For parity with the name of the "Basic API call (JavaScript)", should this `name` attribute be set to "Basic API call (TypeScript)" ?